tree/dataframe/src/RDataFrame.cxx

```diff
@@ -682,6 +682,30 @@ If an instance of [distributed.Client](http://distributed.dask.org/en/stable/api
 provided to the RDataFrame object, it will be created for you and it will run the computations in the local machine
 using all cores available.
 
+### Choosing the number of distributed tasks
+
+A distributed RDataFrame has internal logic to define in how many chunks the input dataset will be split before sending
```

Comment:
Do you want to say a few words about what is typically that default? Otherwise as I reader I don't know if you will already do a good job picking the default or I should bother defining mine.

Reply:
Mmm good point yes